core/include/thread.h

```diff
@@ -11,6 +11,97 @@
  * @ingroup     core
  * @brief       Support for multi-threading
  *
+ * Priorities
+ * ==========
+ *
+ * As RIOT is using a fixed priority scheduling algorithm, threads are
+ * scheduled based on their priority. The priority is fixed for every thread
+ * and specified during the threads creation by the *priority* parameter.
+ *
+ * The lower the priority number, the higher the priority of the thread,
+ * with 0 being the highest possible priority.
+ *
+ * The lowest possible priority is *THREAD_PRIORITY_IDLE - 1*. The exact value
+ * depends on the platforms architecture, e.g. it would be 30 in 32-bit systems,
+ * and 14 in 16-bit systems.
+ *
+ * @note Assigning the same priority to two or more threads is usually not a
+ *       good idea. A thread in RIOT may run until it yields (@ref
+ *       thread_yield) or another thread with higher priority is runnable (@ref
+ *       STATUS_ON_RUNQUEUE) again. Having multiple threads with the same
+ *       priority may make it difficult to determine when which of them gets
+ *       scheduled and how much CPU time they will get. In most applications,
+ *       the number of threads in application is significantly smaller than the
+ *       number of available priorities, so assigning distinct priorities per
+ *       thread should not be a problem. Only assign the same priority to
+ *       multiple threads if you know what you are doing!
+ *
+ * Thread Behavior
+ * ===============
+ * In addition to the priority, flags can be used when creating a thread to
+ * alter the thread's behavior after creation. The following flags are available:
+ *
+ *  - `THREAD_CREATE_SLEEPING`   the newly created thread will be put to sleeping
```

Comment:
Mh… in my editor the diff I proposed was still <100 chars, so well in our line length restrictions.